test-apps/display-test-app/vite.config.ts

```diff
@@ -0,0 +1,131 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { defineConfig, loadEnv, searchForWorkspaceRoot } from "vite";
+import envCompatible from "vite-plugin-env-compatible";
+import browserslistToEsbuild from "browserslist-to-esbuild";
+import viteInspect from "vite-plugin-inspect";
+import { NodeGlobalsPolyfillPlugin } from "@esbuild-plugins/node-globals-polyfill";
+import { NodeModulesPolyfillPlugin } from "@esbuild-plugins/node-modules-polyfill";
+import { externalGlobalPlugin } from "esbuild-plugin-external-global";
+import copy from "rollup-plugin-copy";
+import ignore from "rollup-plugin-ignore";
+import rollupVisualizer from "rollup-plugin-visualizer";
+import { webpackStats } from "rollup-plugin-webpack-stats";
+import * as packageJson from "./package.json";
+import path from "path";
+
+const mode = process.env.NODE_ENV === "development" ? "development" : "production";
+
+// array of public directories static assets from dependencies to copy
+const assets = Object.keys(packageJson.dependencies)
+  .map((pkgName) => {
+    try {
+      // gets dependency path and replaces everything after /lib/ with /lib/public/* to get static assets
+      let pkg = require.resolve(pkgName).replace(/([\/\\]lib[\/\\]).*/, "$1public/*");
+      // use relative path with forward slashes
+      return path.relative(process.cwd(), pkg).replace(/\\/g, '/');
+    } catch { return "undefined"; }
```

Comment:
why not return [undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined)? 


then you could do 
`.filter((path) => path && path.endsWith('lib/public/*'))`
or 
`.filter((path) => path?.endsWith('lib/public/*'))`

Reply:
good idea, I went with the latter and made .map() return `undefined` on error catch